setup.py

```diff
@@ -97,6 +97,7 @@ def run(self):
             'pytest-cov',
             'flaky',
             'nose',  # nose because there are still a few nose.tools imports hanging around
+            'jedi<=0.17.2'
```

Comment:
I think this needs to be fixed in IPython, not here. Pinging @bollwyvl and @Carreau about it.

Reply:
Eventually yes, but the tests were failing without that.

Reply:
I'm aware of this, but I'm swamped and haven't had a chance to look at IPython since mid december, I know there are a bunch of issues and PRs I need o review/fix/merge.

Reply:
@davidbrochart - can we add a comment to the effect here, so that we know later when we can remove the pin?

Reply:
Done in #579